bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -189,37 +140,22 @@ <h3>Hubs</h3>
     <li>hubs.mozilla.com</li>
     <li>reticulum.io</li>
   </ul>
-  
+
   <h3>Localization</h3>
   <ul class="mzp-u-list-styled">
-    <li>l10n.mozilla.org</li>
     <li>pontoon.mozilla.org</li>
   </ul>
 
   <h3>Observatory</h3>
   <ul class="mzp-u-list-styled">
-    <li>http-observatory.security.mozilla.org</li>
     <li>observatory.mozilla.org</li>
     <li>tls-observatory.services.mozilla.com</li>
   </ul>
 
-  <h3>Payment Subscription</h3>
-  <ul class="mzp-u-list-styled">
-    <li>prod.fxa.mozilla-subhub.app</li>
-    <li>subscriptions.firefox.com</li>
```

Comment:
We definitely still use subscriptions.firefox.com, and we take people's money and credit cards on it. If we're not willing to stand behind this one that's a bad sign.

I don't know about the mozilla-subhub thing -- no DNS entry, maybe that was an earlier implementation, different partner site (e.g. maybe a custom domain used by https://www.subhub.com/)?  Our current subscriptions seem to run on a mozilla GCP domain and talk to stripe and paypal on the back-end, so the mozilla-subhub.app looks fine to delete.